server/src/models/Trail.js

```diff
@@ -0,0 +1,22 @@
+const Model = require("./Model")
+
+class Trail extends Model {
+    static get tableName(){
+        return "trails"
+    }
+
+    static get jsonSchema(){
+        return {
+            type: "object",
+            required: ["trailName", "trailLength", "trailLocation"],
+            properties: {
+                trailName: {type: "string"},
+                trailLength: {type: ["string", "integer"]},
+                trailDescription: {type: "string"},
+                trailLocation: {type: "string"},
+                estimateTime: {type: ["string", "integer"]}
+            }
+        }
+    }
+}
```

Comment:
Again looks like an indentation of 4.  Make sure everyone has their tab space settings on 2